docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,92 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+When using DDSes, data is sequenced and stored within the Fluid container to achieve synchronized shared state. For scenarios that involve shared persisted data, DDSes provide an effective way to communicate data so that it is retained in the container. However, there are many scenarios where we need to data communicate that is short-lived, in which the ordering and storage of said information would be wasteful and unnecessary.
+
+In contrast, the data that is communicated via signals is not retained in the container, which makes it an appropriate channel for transmitting transient data. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol. Signals are not guaranteed to be ordered on delivery relative to other signals and ops, but they still provide a useful communication channel for impermanent and short-lived data.
```

Comment:
Looking at it now, I agree that the analogy could be confusing to people unfamiliar with networking protocols and I think it's removal would be justified. 

To add more context, this was the analogy used to help describe the project to me and I did find it to be helpful since I was already familiar with the two networking protocols. The analogous part of the comparison is that UDP and signals are more lossy, faster, not guaranteed delivery, simpler, and have that "fire-and-forget" usage. TCP and DDSes guarantee more and are therefore more complicated in their implementation and usage in general.

Reply:
Agreed! Lets keep this simple. The networking analogy can get confusing. One thing we are trying to do with docs is that we make it easy for all levels of developers be able to understand these concepts better. I think you already explained it well with words like lossy, guarantee etc.